packages/terra-tabs/src/StructuralTabs.scss

```diff
@@ -21,18 +24,37 @@
       }
 
       .tab-menu.is-active {
-        background-size: 0 0;
+        background-size: var(--terra-tabs-collapsed-tab-menu-active-background-size, 0 0);
       }
 
       .tab-menu:hover {
-        background-size: 0 0;
+        background-size: var(--terra-tabs-collapsed-tab-menu-hover-background-size, 0 0);
       }
     }
 
     .collapsible-tabs-container {
       border-bottom-color: var(--terra-tabs-structural-tab-bar-border-color, #dddedf);
       border-bottom-style: solid;
       border-bottom-width: var(--terra-tabs-structural-tab-bar-border-width, 1px);
+
+      // Only target the tabs when in collapsible-tabs-container. When it is collapsed, the menu shouldn't have the stripe
+      .tab,
+      .tab-menu {
+        position: var(--terra-tabs-structural-position, static);
```

Comment:
This variable is missing the `-tab-` substring, or the ones below have an extra `-tab-`

Reply:
hmm, the existing ones even further below are
`--terra-tabs-structural-background-image`
so I'm gonna go with I have an extra `-tab` for my before content variables. So i'll leave this one the same, and update the ones below to 
`--terra-tabs-structural-before-content-background`
`--terra-tabs-structural-before-content-height`

sound good?

Reply:
Sounds good to me